Make.config

```diff
@@ -66,6 +66,10 @@ MIN_MONO_VERSION=5.14.0.136
 MAX_MONO_VERSION=5.14.99
 MIN_MONO_URL=https://xamjenkinsartifact.azureedge.net/build-package-osx-mono/2018-04/111/07c8f25fe536dbb7e244f965aa3f9a871f41e953/MonoFramework-MDK-5.14.0.136.macos10.xamarin.universal.pkg
 
+# Minimum Mono version for Xamarin.Mac apps using the system mono
+MIN_XM_MONO_VERSION=5.14.0.136
```

Comment:
Why are we not simply using the `MIN_MONO_VERSION`? Do you expect the min requirement to differ between XI and XM (not the case today)? If one (XI/XM) has a higher requirement than the other, we'll have to use the higher versioned mono anyway. So I'm not sure why we need two variables.

Reply:
Because they're for different things.

* MIN_MONO_VERSION: the minimum mono version to build (on our machines) XI and XM.
* MIN_XM_MONO_VERSION: the minimum mono version to build/run XM apps that use system mono (on customers machines).

These can be wildly different.